fusillade/clouddirectory.py

```diff
@@ -663,10 +667,12 @@ def delete_policy(self, policy_ref: str) -> None:
             DirectoryArn=self._dir_arn,
             ObjectReference={'Selector': policy_ref})
 
-    def delete_object(self, obj_ref: str) -> None:
+    def delete_object(self, obj_ref: str, traverse=False) -> None:
```

Comment:
Nit: would `delete_children` be a more descriptive name for the `traverse` parameter?

When an object is deleted, is it possible to lose references to the objects children?

Reply:
delete_children is better. I'll change.

Reply:
> When an object is deleted, is it possible to lose references to the objects children?
Deletion starts from the leaves to the branches so there will be no orphaned children.